pkg/specgen/generate/namespaces.go

```diff
@@ -300,6 +300,16 @@ func namespaceOptions(s *specgen.SpecGenerator, rt *libpod.Runtime, pod *libpod.
 			val = fmt.Sprintf("slirp4netns:%s", s.NetNS.Value)
 		}
 		toReturn = append(toReturn, libpod.WithNetNS(portMappings, expose, postConfigureNetNS, val, nil))
+	case specgen.Pasta:
+		portMappings, expose, err := createPortMappings(s, imageData)
+		if err != nil {
+			return nil, err
+		}
+		val := "pasta"
+		if s.NetNS.Value != "" {
+			val = fmt.Sprintf("pasta:%s", s.NetNS.Value)
+		}
```

Comment:
this is not required, I do not understand why this is done for slirp because s.NetNS.Value must be empty anyway.

Reply:
Dropped, tested, not really needed. I left the case for slirp4netns, as I'm not sure enough of the logic.